content/tips-on-applying-for-teacher-training.md

```diff
@@ -0,0 +1,179 @@
+interviews.
+
+## Browse courses
+
+You can [search for most postgraduate teacher training courses on GOV.UK](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+You can apply to up to 3 courses in total, and then accept one offer.
+
+Many people do not get in the first time, but there’s no need to wait until the
+next year to apply again.
+
+You can rework your application and apply to further courses one at a time.
+
+[Attend an event](/events) to get your course questions answered.
+
+If you do not find the right course for you on GOV.UK, there are [other ways to
+train](/ways-to-train).
+
+## Questions to ask your training provider
+
+Every training provider and training programme is different.
+
+Note down any questions that come to mind while browsing courses.
+
+You may want to ask potential training providers questions, such as:
+
+1. How are academic work and practical teaching arranged?
+2. Who will my mentor be and how often will I meet with them?
+3. Which schools and how many schools will I teach in?
+4. Will I get any other qualifications aside from qualified teacher status
+   (QTS), and what are the benefits of that?
+5. Do you advise me to [get school experience](https://schoolexperience.education.gov.uk/)?
+6. Will you support me to find my first teaching job?
+7. How long does the training take?
+
+## When to apply
+
+Courses usually open for applications in October for entry the following year
+(usually starting in September, but sometimes January).
+
+Teacher training providers allocate places as people apply through the year,
+and courses stay open until they are full.
+
+If you’re keen to join a popular course it’s wise to apply as soon as you’re
+ready rather than putting it off.
+
+## What to expect: the application form
+
+When you apply you’ll need to give details about:
+
+* your qualifications, including your GCSEs and A levels (or equivalents) and degree
+* your work history or unpaid experience
+* why you want to teach
+* why you’re suited to teach a particular subject or age group
+
+You can also share whether you need any adjustments during the application
+process or on the course - for example, if you’re disabled.
+
+You’ll be encouraged to declare any potential safeguarding issues such as
+criminal convictions. Training providers can advise you whether it’ll affect
+your application.
+
+## Prepare your application
+
+You can [find and apply to most postgraduate teacher training courses on GOV.UK](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+
+If you do not find what you were looking for, there are [other ways to train](/ways-to-train).
+
+Before you start, it’s particularly important to think about your referees and
+personal statement.
+
+Some people do not get on the course they want because their application has
+not convinced the admissions team that they’re right for it.
+
+Talk to an experienced teacher for help preparing your application.
```

Comment:
looking at Rose's original doc, I think this line is supposed to be a link to the adviser service